app/views/spree/admin/orders/_assemblies.html.erb

```diff
@@ -1,62 +1,54 @@
 <% if order.line_items.assemblies.any? %>
-  <fieldset class="no-border-bottom">
-    <legend align="center" class="" data-hook="">
-      <%= Spree.t(:product_bundles) %>
-    </legend>
-  </fieldset>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h1 class="panel-title">
+        <%= Spree.t(:product_bundles) %>
+      </h1>
+    </div>
+    <table class="table table-bordered product-bundles" data-hook="">
+      <thead>
+        <th colspan="2"><%= Spree.t(:item_description) %></th>
+        <th><%= Spree.t(:price) %></th>
+        <th><%= Spree.t(:quantity) %></th>
+        <th><%= Spree.t(:total) %></th>
+        <th class="orders-actions actions" data-hook="admin_order_form_line_items_header_actions"></th>
+      </thead>
 
-  <table class="product-bundles" data-hook="">
-    <colgroup>
-      <col style="width: 10%;" />
-      <col style="width: 30%;" />
-      <col style="width: 15%;" />
-      <col style="width: 15%;" />
-      <col style="width: 15%;" />
-      <col style="width: 15%;" />
-    </colgroup>
+      <% order.line_items.assemblies.each do |item| %>
+        <tbody>
+          <tr class="" data-item-quantity="<%= item.quantity %>">
+            <td class="item-image"><%= mini_image(item.variant) %></td>
+            <td class="item-name">
+              <%= item.product.name %> - <%= item.variant.sku %> <br />
+              <%= "(" + variant_options(item.variant) + ")" unless item.variant.option_values.empty? %>
```

Comment:
Same here (i.e. delegate)

Also, what do think about changing the double negatives to `if... any?`